mongodbatlas/resource_mongodbatlas_encryption_at_rest.go

```diff
@@ -271,10 +267,8 @@ func flattenAWSKMS(m *matlas.AwsKms) map[string]interface{} {
 		return map[string]interface{}{
 			"enabled":                cast.ToString(m.Enabled),
 			"access_key_id":          m.AccessKeyID,
-			"secret_access_key":      m.SecretAccessKey,
 			"customer_master_key_id": m.CustomerMasterKeyID,
 			"region":                 m.Region,
-			"role_id":                m.RoleID,
```

Comment:
I'm not sure I understand why these two values are being removed?

Reply:
Those parameters doesn't appear in API response and in docs it doesn't appear either in response, if I set those values, it will always appear "changes" since the result its empty like "" > "values"

Reply:
Ah! That makes sense, I didn't expect role_id to not be returned but if you verified the actual return matches docs then we are good.

Reply:
I tested again, it seems with `role_id` doesn't give changes only with secret key. so I put it back the role id